.github/workflows/codspeed.yml

```diff
@@ -16,7 +16,6 @@ jobs:
         with:
           python-version: '3.12'
       - run: python -m pip install .[tests] pytest-codspeed 'numpy<2'
-      - run: python -c "from dkist.data.sample import download_all_sample_data; download_all_sample_data()"
```

Comment:
I don't think we need to remove this. This means that the data get's downloaded in it's own step and then the rest of the benchmarks run can be offline.

Reply:
I thought this was what was causing the problems though? If we bring this back then we probably don't need the remote_data mark on at least some of the ones I've added it to, because it won't need to download the data.

Reply:
It should have the remote data mark, because it wont work without the sample data (i.e if someone runs it locally). But there's no reason why we can't also pre-download the sample data in a separate step of the workflow.

Reply:
Also to make the benchmarks run with remote data, add `--remote-data=any` to the pytest line at the end of this file.